docs/how-to/h-upgrade.md

```diff
@@ -0,0 +1,125 @@
+# How to upgrade between minor versions
+
+> **Note** This feature is available on Charmed Kafka and Charmed ZooKeeper from revisions XXX and YYY, respectively. Upgrade from previous versions is **not supported**.
```

Comment:
Are the exact XXX/YYY revisions known yet?

Reply:
Not yet...they will be known tonight (hopefully) and surely tomorrow when the final release will be done :D